From 5b1263fc57530948df7a7f75c27567b8a85ab545 Mon Sep 17 00:00:00 2001
From: uros <uros@138bc75d-0d04-0410-961f-82ee72b054a4>
Date: Tue, 1 Oct 2013 15:02:22 +0000
Subject: [PATCH 04/18] Fix doc/gcc.texi for makeinfo in texinfo-6.4

	* doc/gcc.texi (titlepage): Use @uref and http:// prefix for website.
	Use @email for email addresses.

git-svn-id: svn+ssh://gcc.gnu.org/svn/gcc/trunk@203065 138bc75d-0d04-0410-961f-82ee72b054a4
---
 gcc/doc/gcc.texi | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/gcc/doc/gcc.texi b/gcc/doc/gcc.texi
index 808b8207d16..d6cfd4aa70d 100644
--- a/gcc/doc/gcc.texi
+++ b/gcc/doc/gcc.texi
@@ -83,11 +83,11 @@ This file documents the use of the GNU compilers.
 Published by:
 @multitable @columnfractions 0.5 0.5
 @item GNU Press
-@tab Website: www.gnupress.org
+@tab Website: @uref{http://www.gnupress.org}
 @item a division of the
-@tab General: @tex press@@gnu.org @end tex
+@tab General: @email{press@@gnu.org}
 @item Free Software Foundation
-@tab Orders:  @tex sales@@gnu.org @end tex
+@tab Orders:  @email{sales@@gnu.org}
 @item 51 Franklin Street, Fifth Floor
 @tab Tel 617-542-5942
 @item Boston, MA 02110-1301 USA


